README.md

```diff
@@ -40,6 +40,91 @@ Please follow the instructions given in the following link to connect Azure Boar
 
 [Connect Azure Boards to Github](https://docs.microsoft.com/en-us/azure/devops/boards/github/connect-to-github?view=azure-devops)
 
-## 7. Contact us
+
+## 7. Configure Data Share
```

Comment:
Osama, you only need to provide a brief here that the user can setup datashare and just had to add a link to the doc you already added in the public/customer folder

Reply:
ok